statistics/handle/dump.go

```diff
@@ -282,6 +282,9 @@ func (h *Handle) LoadStatsFromJSON(ctx context.Context, is infoschema.InfoSchema
 			}
 		}
 	}
+	if skipUpdate {
+		return nil
+	}
```

Comment:
I will prefer just extract all the codes before to a function (Say, `LoadStatsFromJSONNoUpdate`), and call this function at the origin `LoadStatsFromJSON`. So the new function signature can be kept unchanged hence there is no need to modify so lines...

Reply:
LGTM